csharp/Platform.Ranges/Range[T].cs

```diff
@@ -58,89 +70,144 @@ public Range(T minimum, T maximum)
         }
 
         /// <summary>
-        /// <para>Presents the Range in readable format.</para>
-        /// <para>Представляет диапазон в удобном для чтения формате.</para>
+        /// <para>Presents the <see cref="Range{T}"/> in readable format.</para>
+        /// <para>Представляет <see cref="Range{T}"/> в удобном для чтения формате.</para>
         /// </summary>
-        /// <returns><para>String representation of the Range.</para><para>Строковое представление диапазона.</para></returns>
+        /// <returns>
+        /// <para>String representation of the <see cref="Range{T}"/>.</para>
+        /// <para>Строковое представление <see cref="Range{T}"/>.</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public override string ToString() => $"[{Minimum}..{Maximum}]";
 
         /// <summary>
-        /// <para>Determines if the provided value is inside the range.</para>
-        /// <para>Определяет, находится ли указанное значение внутри диапазона.</para>
+        /// <para>Determines if the <see cref="Range{T}"/> contains the <paramref name="value"/>.</para>
```

Comment:
`Determines whether the <see cref="Range{T}"/> contains the <paramref name="value"/>.`